vanilla/scss/styles.scss

```diff
@@ -70,3 +70,28 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+// Toc custom styles
+.p-table-of-contents__nav > ul > li:first-child > a {
```

Comment:
Because everything is custom and very specific it would be good to have a comment in the code above every part of CSS to describe what it's doing and why, so something like `// hide first link in TOC which contains title of current page`, etc (for the others below it that implement indentation as well).